redaxo/src/addons/cronjob/plugins/article_status/lang/de_de.lang

```diff
@@ -1 +1,3 @@
 cronjob_article_status = Artikel-Status
+cronjob_article_reset_date = Artikel-Status nur einmalig ändern
+cronjob_article_reset_date_info = Mit dieser Option wird der Status eines Artikels nur einmal geändert und das "online_from"-Datum bzw. "online_to"-Datum<br>in den Metadaten des Artikels anschließend gelöscht.
```

Comment:
Diese Bezeichnung finde ich noch nicht optimal. Da würde ich denken, warum sollte es überhaupt mehrfach geändert werden? (Ich weiß aber, wie du zu der Bezeichnung kamst)
Vielleicht eher was in die Richtung "Datumsfelder nach Statusänderung leeren".

Auch in der Langbeschreibung finde ich dieses "einmalig geändert" eher verwirrend (wenn man deinen Hintergrund nicht kennt, wieso es bei dir mehrfach geändert wurde).
Wenn man den Aspekt erwähnen möchte, müsste man es denke ich noch genauer machen. Also etwas wie 
"... werden geleert, sodass nach anschließender manuelle Statusänderung, diese nicht erneut durch den Cronjob zurückgewechselt werden" (oder irgendwie so in der Art).
Für mich würde aber auch nur das "was" reichen, also dass die Felder geleert werden.